packages/hop-node/src/watchers/classes/BaseWatcher.ts

```diff
@@ -47,6 +47,7 @@ class BaseWatcher extends EventEmitter implements IBaseWatcher {
   dryMode: boolean
   tag: string
   prefix: string
+  syncIndex: number
```

Comment:
nit pick: the default assignment should be here instead of in the constructor

Reply:
Fix 1b4fc481ea7398f99fbd7364d4ae3a6a38eaf4dc